Neos.Media/Classes/Domain/Model/Adjustment/ResizeImageAdjustment.php

```diff
@@ -237,6 +237,12 @@ public function getMinimumWidth(): ?int
      */
     public function setRatioMode(string $ratioMode): void
     {
+        if ($ratioMode !== ImageInterface::RATIOMODE_INSET &&
+            $ratioMode !== ImageInterface::RATIOMODE_OUTBOUND
+        ) {
+            throw new \InvalidArgumentException('Invalid mode specified', 1574686876);
```

Comment:
Not sure how likely this case is, but debugging would be easier with sth like:
```php
$supportedModes = [ImageInterface::RATIOMODE_INSET, ImageInterface::RATIOMODE_OUTBOUND];
if (!in_array($ratioMode, $supportedModes, true)) {
    throw new \InvalidArgumentException(sprintf('Invalid mode "%s" specified, supported modes are: "%"', $ratioMode, implode('", "', $supportedModes)), 1574686876);
}
```
(untested)

Reply:
Nice! But why not a committable suggestion?